distribution/ddf-common/src/main/resources/etc/blacklisted.properties

```diff
@@ -0,0 +1 @@
+pax-jetty;type=feature;range=[9.3,9.3.21]
```

Comment:
Could we add 
jetty;type=feature
pax-jetty-http2;type=feature
to clean up uninstalled jetty features as well?